java/openmldb-taskmanager/src/test/java/com/_4paradigm/openmldb/taskmanager/TestJobIdGenerator.java

```diff
@@ -0,0 +1,50 @@
+package com._4paradigm.openmldb.taskmanager;
+
+import com._4paradigm.openmldb.taskmanager.dao.JobIdGenerator;
+import org.testng.Assert;
+import org.testng.annotations.Ignore;
+import org.testng.annotations.Test;
+import java.util.concurrent.Callable;
+import java.util.concurrent.CopyOnWriteArrayList;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+
+public class TestJobIdGenerator {
+    public CopyOnWriteArrayList<Integer> jobIdSet = new CopyOnWriteArrayList<>();
+
+    public class ChildThread implements Callable<Void> {
+        int iterNum = 100000;
+        public Void call() throws Exception {
+            for (int i=0; i < iterNum; i++) {
+                int jobId = JobIdGenerator.getUniqueId();
+                if (jobIdSet.contains(jobId)) {
```

Comment:
where is `jobIdSet` updated?

Reply:
jobIdSet should be added before returning.